### DISCUSSION TITLE
Customize User table

### DISCUSSION THREAD

**Discussion by dkunzman77** [strapi/strapi#7460] [Legacy Topics]
I would like to customize the User table with custom field that I update programatically.  I tried to generate a new model named users and the application wouldn't start error is "Field "DeleteUserPayload.user" can only be defined once.  assertValidSDL line 108.  
How could I do this?

**Comment by derrickmehaffy** [comment]
You aren't currently able to have multiple models with the same name (`user` in this case) but you can easily extend the users model by following this: https://strapi.io/documentation/v3.x/getting-started/troubleshooting.html#how-do-i-customize-a-plugin

Or simply by adding the fields in the Content-Type builder on the admin panel.

**Comment by dkunzman77** [comment]
Thanks you for answering my question.  Just to be clear I need to remove the content from the current email fields in the strapi-administrators and the in the users-permissions-user tables making it encrypted data at rest (in the file).  If I understand you correctly I can create an extensions model to do that?  I assume the best documentation now is the source code strapi-generate-admin is that correct?

**Reply by derrickmehaffy** [reply]
You will have to do some extensions yes, this is possible for users-permissions but not currently possible with the `strapi-admin` package without forking the entire package. We have an open feature request for this here: https://github.com/strapi/strapi/issues/4838
